### PR TITLE
PLAT-124534: Support new RTL scroller coordinate of Chrome 85

### PR DESCRIPTION
**Pull request by jeonghee27**
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [ ] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
ScrollThumb in horizontalScrollbar is located abnormal position when clicking scrolltrack  with RTL.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Support new RLT scroller coordinate of Chrome 85 (https://www.chromestatus.com/feature/5759578031521792)
- Horizontal (or vertical) scroll position will be zero at its initial position and negative when scrolled leftward (or upward).

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-124534

### CODE REVIEW

File: packages/ui/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ The following is a curated list of changes in the Enact ui module, newest change
 
 ## [unreleased]
 
+### Fixed
+
+- `ui/Scroller`, `ui/VirtualList`, and `ui/VirtualGridList` to scroll correctly on latest Chrome
```
Comment (0x64) [inline packages/ui/CHANGELOG.md:9]:
```suggestion
- `ui/Scroller`, `ui/VirtualList`, and `ui/VirtualGridList` to scroll correctly on Chrome 85 or higher in RTL locales
```

Comment (MikyungKim) [reply packages/ui/CHANGELOG.md:9]: `ui/Scroller` and `ui/VirtualList` would be fine.

Comment (jeonghee27) [reply packages/ui/CHANGELOG.md]: I fixed it

**Review by 0x64 — APPROVED** [review]
:lgtm:

**Comment by 0x64** [comment]
I changed the merge target to `release/3.4.10`.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**